src/commands/react_native_xcode.rs

```diff
@@ -274,15 +274,19 @@ pub fn execute(matches: &ArgMatches<'_>) -> Result<(), Error> {
         processor.rewrite(&[base.parent().unwrap().to_str().unwrap()])?;
         processor.add_sourcemap_references()?;
 
-        let release = api.new_release(
+        let default_release_name = format!(
+            "{}@{}+{}",
+            plist.bundle_id(),
+            plist.version(),
+            plist.build()
+        );
+
+        let release_name = env::var("SENTRY_RELEASE_NAME").unwrap_or(default_release_name);
```

Comment:
```suggestion
        let release_name = env::var("SENTRY_RELEASE").unwrap_or(default_release_name);
```
Let's call it `SENTRY_RELEASE`.

Reply:
done